wallet/wallet.go

```diff
@@ -2554,6 +2560,86 @@ func (w *Wallet) GetTransaction(txHash chainhash.Hash) (*GetTransactionResult,
 	if err != nil {
 		return nil, err
 	}
+
+	// If the transaction was not found in the transaction store, we need to
+	// build the TransactionSummary from the backend RPC response.
+	if summary == nil {
+		chainClient, err := w.requireChainClient()
+		if err != nil {
+			return nil, err
+		}
+
+		// Get the transaction data from the backend endpoint.
+		var txResult *btcjson.TxRawResult
+		switch client := chainClient.(type) {
+		case *chain.RPCClient:
+			txResult, err = client.GetRawTransactionVerbose(&txHash)
+			if err != nil {
+				return nil, err
+			}
+		case *chain.BitcoindClient:
```

Comment:
What about Neutrino? Those clients would currently panic on the `txResult.Hex` access below.